docs/user-guides/capacity-plannig.md

```diff
@@ -23,21 +23,27 @@ For example, if you want to insert 1 million vectors with 900 dimensions and the
 (900 × 32 + 256 ) × 1,000,000 × 3 = 8,7168,000,000 (bit) = 10.896 (GB)
 ```
 
-
 It is just the minimum required RAM.
-Considering the margin of RAM capacity, the minimum RAM capacity should be less than 60% of the actual RAM capacity. 
+Considering the margin of RAM capacity, the minimum RAM capacity should be less than 60% of the actual RAM capacity.
 Therefore, the actual minimum RAM capacity will be:
 
 ```bash
 8,7168,000,000 (bit) / 0.6 = 145,280,000,000 (bit) = 18.16 (GB)
 ```
 
+<div class="warn">
+Memory usage is not enough in the minimum required RAM.<BR>
```

Comment:
In many production operation cases, memory usage can be unstable with minimal RAM required. <BR>